src/main/java/it/gov/pagopa/arc/service/CustomAuthenticationSuccessHandler.java

```diff
@@ -20,28 +21,59 @@ public class CustomAuthenticationSuccessHandler implements AuthenticationSuccess
   private final ObjectMapper objectMapper;
   private final TokenStoreService tokenStoreService;
   private final JWTConfiguration jwtConfiguration;
+  private final AuthService authService;
 
   CustomAuthenticationSuccessHandler(
       AccessTokenBuilderService accessTokenBuilderService,
       ObjectMapper objectMapper,
       JWTConfiguration jwtConfiguration,
-      TokenStoreService tokenStoreService){
+      TokenStoreService tokenStoreService,
+      AuthService authService){
```

Comment:
the cf list has not usage outside of this component...
I will not expose it adding a new method on the AuthService...
let's just inject here the Set

ps: let's explicitly add the `white-list` property, bind to the ENV var, inside the application.yml
```suggestion
      @Value("${white-list}") Set<String> cfWhitelist){
```